Fixup MathCAT settings (#19227)
nvaccess/nvda: user_docs/en/userGuide.md

##### Speech Options {#MathCATSpeechOptions}

> ###### Generate speech for {#MathSpeechImpairment}

Reviewer: ```suggestion
###### Generate speech for disability {#MathSpeechImpairment}
```

Changes the relative speech rate.
The change is a percentage speed change from the standard speech engine's rate.

> In other words, `100` means the reading rates of text and mathematical content are the same, whereas `50` means that mathematical content is read at half the speed of other content.

Reviewer: ```suggestion
`100` means the reading rates of text and math content are the same, whereas `50` means that math content is read at half the speed of other content.
```

###### Make a sound when starting/ending math speech {#MathSpeechSound}

> 

Reviewer: please restore this

Reviewer: this hasn't been restored yet

###### Copy math as {#MathNavCopyAs}

> This option selects the format of mathematical content copied to the clipboard.

Reviewer: ```suggestion
This option selects the format of math content copied to the clipboard.
```
